apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -230,6 +230,108 @@ class SbbolFintechApi extends SbbolRequestApi {
     get apiPrefix () {
         return '/fintech/api'
     }
+    /**
+     * @typedef ClientAccounts
```

Comment:
Could you define type for one entity, like `ClientAccount` and specify an array `ClientAccount[]` here as a return type

Reply:
I think this should be removed. Since no one will update it, and this is someone else's api. If you need information, it is better to look at the current SberBusiness documentation. If I'm wrong and it's better to describe the type here, I can do

Reply:
OK, decide yourself on following criteria: does this doc saves time for someone, who sees this code first time.

Reply:
Since, it is not only `{ accounts }`, but `{ inn, accounts }`, it will be more informative, if you rename it accordingly, like `SbbolClientAccountsInfo`.

You can define docs in a following way:

```js
/**
 * @typedef SbbolClientAccountsInfo
 * @property {String} inn
 * @property {SbbolClientAccount[]} accounts
 *
*/
```

And define account schema separately:

```js
/**
 * @typedef SbbolClientAccount
 * @property {String} number
 * @property {String} name
 * …
 */